research/models/group.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from django_extensions.db.models import TitleDescriptionModel, TimeStampedModel
+
+
+class Group(TitleDescriptionModel, TimeStampedModel):
+    study = models.ForeignKey("research.Study", on_delete=models.CASCADE)
```

Comment:
> all of the **models**

i.e. groups? 

> to be **updated**

to be deleted, yes. 
If we're deleting the study itself I figured there's no need to keep its groups. I don't think PROTECT would make sense because it seems like on overkill to have to delete each group individually in order to delete the study.

Reply:
but do groups have any significance when not associated with a study?

Reply:
No significance if not associated with a study, that's why it's cascading.